### DISCUSSION TITLE
Media materie

### DISCUSSION THREAD

**Discussion by sempliciniluigi89** [iisgiua/giuaschool#333] [Idee e proposte]
Buongiorno , sarebbe possibile far apparire accanto ad ogni materia la media matematica? É molto importante per noi tutti ,ovviamento sia dal lato docente che genitore/alunno . La stessa richiesta feci per il progetto Lampschool e loro in 2 giorni massimo implementarono questa cosa , che era un vero e proprio quadratino affiancato ad ogni materia riportante la media della materia , se questa era sotto la sufficienza ,allora era rossa ,altrimenti era verde .Poi in alto ,sotto il nome o accanto al nome dello studente c'era la media complessiva di tutte le materie

**Comment by trinko** [comment]
La segno come richiesta
